generate.ts

```diff
@@ -119,6 +125,15 @@ function generate() {
 
   function getArgsInfo(args: z.infer<typeof TypeArgs>, required: boolean, alias: boolean, typeString?: string): Arg {
     if (Array.isArray(args)) {
+      const isUnion = args.every(arg => arg in doc.types);
+      if(isUnion) {
```

Comment:
nit: i think the above check can be inlined here

Reply:
you mean not store the `isUnion` const and put the condition right in the `if`?

Reply:
yes